### PR TITLE
Compatibilize SDK with the new API.

### PR DESCRIPTION
**Pull request by lcbm**
This PR introduces the following changes:

- Compatibilize HTTP operations;
- Compatibilize createUser() method;
- Add createToken() method;
- Remove methods which aren't compatible with the new API;

### CODE REVIEW

File: src/Client.js [hunk 1]
```diff
@@ -1,68 +1,18 @@
...
-import url from 'url';
-
-function throwError(message) {
-  const error = new Error(message);
-  throw error;
-}
+import HTTP from './network/HTTP';
 
 export default class Client {
-  constructor({
```
Comment (netoax) [inline src/Client.js:10]: The current constructor is already suitable for our needs and it does goods verifications, why did you replace it?

File: package.json [hunk 2]
```diff
@@ -8,8 +8,7 @@
 	"scripts": {
 		"lint": "eslint .",
 		"build": "rollup -c",
-		"prepublishOnly": "npm run build",
-		"test": "jest"
```
Comment (netoax) [inline package.json:12]: What do you think about removing the test files in another commit? I think it'd easy to revert it in the future.

**Review by netoax — CHANGES_REQUESTED** [review]
Great PR, thanks!
I did just a few comments.

File: package.json (hunk 2 above)
Comment (lcbm) [reply package.json:12]: Done :smile:

File: src/Client.js (hunk 1 above)
Comment (lcbm) [reply src/Client.js:10]: Oppsie, hehe... done :smile: 
A didn't quite get the reason for `path`, but let me know if I should add it :smile:

**Review by netoax — APPROVED** [review]

**Pull request merged**